mmv1/api/resource.go

```diff
@@ -1618,9 +1618,13 @@ func (r Resource) TestExamples() []resource.Examples {
 }
 
 func (r Resource) VersionedProvider(exampleVersion string) bool {
-	vp := r.MinVersion
+	var vp string
 	if exampleVersion != "" {
 		vp = exampleVersion
+	} else if r.MinVersion == "" {
```

Comment:
follows the old ruby logic more closely. It didn't matter for GA/Beta, but it makes diffs in private. See the old logic spread out between:
https://github.com/GoogleCloudPlatform/magic-modules/blob/legacy-ruby/mmv1/api/resource.rb#L606
and
https://github.com/GoogleCloudPlatform/magic-modules/blob/legacy-ruby/mmv1/templates/terraform/examples/base_configs/test_file.go.erb#L55
and
https://github.com/GoogleCloudPlatform/magic-modules/blob/legacy-ruby/mmv1/templates/terraform/examples/base_configs/test_file.go.erb#L77